packages/templates/cra-template-web-viewer/template/src/index.tsx

```diff
@@ -27,11 +28,18 @@ if (!process.env.IMJS_AUTH_CLIENT_REDIRECT_URI) {
   );
 }
 
+Auth.initialize({
```

Comment:
can this be done in App.tsx like before?

Reply:
Unfortunately, no. It won't be hit on the callback since App.tsx loads only if not the callback route, and the configuration won't be defined.